solr/views.py

```diff
@@ -165,9 +177,9 @@ def post(self):
                 {'error': 'You can only pass one content stream.'}), 400
 
         if 'fq' not in query:
-            query['fq'] = [u'{!bitset}']
-        elif len(filter(lambda x: '!bitset' in x, query['fq'])) == 0:
-            query['fq'].append(u'{!bitset}')
+            query['fq'] = '{!bitset}'
+        elif '{!bitset}' not in query['fq']:
```

Comment:
this wouldn't match (a syntactically correct) '{ !bitset }' or `{!bitset compression=gzip}'